src/ReportManager/DetailsSection/SchemaForm/utils/useSchemaValidations/index.js

```diff
@@ -0,0 +1,34 @@
+import { useCallback, useMemo } from 'react';
```

Comment:
**Related to AJV validations**
Hook that receives an schema, memoizes the AJV compilation of its validate function and returns a `runValidations` memoized method that validates a form data object against the schema and returns a flat errors object mapping an internationalized message per error by the id of the field:
```
const runValidations = useSchemaValidations(schema);
...
const errors = runValidations(formData);
if (errors) {
  // errors is a flat object like { fieldId: 'Meaningful internationalized message with the error!' }
} else {
  // No errors!
}
```